examples/wursthall/models/src/customer_details.py

```diff
@@ -50,7 +50,7 @@ def execute(
         for _ in range(random.choice(range(10, 20))):
             customer_details.append(
                 CustomerDetails(
-                    id=faker.uuid4(),
+                    id=str(faker.uuid4()),
```

Comment:
FYI this change was triggered due to a mypy [error](https://app.circleci.com/pipelines/github/TobikoData/sqlmesh/12841/workflows/1a26c970-de75-46a1-8580-0991362ac09c/jobs/94167), probably caused by the latest Faker [release](https://github.com/joke2k/faker/releases/tag/v25.0.1).

I verified that the uuid is a string value, but I went ahead and used `str` here to enforce this instead of `t.cast`.

```python
>>> import faker
>>> type(faker.Faker().uuid4())
<class 'str'>
```